.github/workflows/nodejs.yml

```
@@ -0,0 +1,85 @@
+name: firebase deploy
+
+on:
+  push:
+    paths:
+      - "_src/**"
```

```suggestion
      - "lib/**"
```

This app was started before we standardized on `_src`